platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```

Comment:
Are we certain we should be removing the earlier years on these?

Reply:
that has been created by (I assume) spotless

Reply:
@nathanklick what is the wished format?